index.js

```diff
@@ -129,17 +129,71 @@ document.addEventListener("DOMContentLoaded", function () {
   }
 });
 
-//color change//
 const prevBtn = document.querySelector('.js-prev');
 const nextBtn = document.querySelector('.js-next');
 const ellipse = document.querySelector('.js-food-spin-ellipse');
 const button = document.querySelector('.js-food-spin-order-button');
 const price = document.querySelector('.js-food-spin-price');
 const title = document.querySelector('.js-food-spin-title');
 const description = document.querySelector('.js-food-spin-description');
-const items = [...document.querySelectorAll('.js-food-spin-item')];
+const pagination = document.querySelector('.js-carousel-food-spin-pagination');
+const menuItems = [...document.querySelectorAll('.js-food-spin-item')];
+const ItenSubmenu = [...document.querySelectorAll('.js-food-spin-submenu')];
+const dishesPlatos = [...document.querySelectorAll('.js-carousel-food-spin-items')];
+
+let currentIndex = 0;
+
+function changeInterfaceColors(dishInfo) {
+  const colorSet = {
+    ellipseColor: dishInfo.getAttribute('ellipseColor'),
+    shadowColor: dishInfo.getAttribute('shadowColor'),
+    buttonColor: dishInfo.getAttribute('buttonColor'),
+    hoverColor: dishInfo.getAttribute('hoverColor'),
+  };
+
+  ellipse.style.setProperty('--bg-color', colorSet.ellipseColor);
+  pagination.style.setProperty('--bg-orange', colorSet.buttonColor);
+  price.style.setProperty('--bg-orange', colorSet.buttonColor);
+  prevBtn.style.setProperty('--bg-color', colorSet.buttonColor);
+  prevBtn.style.setProperty('--shadow-bg', colorSet.shadowColor);
+  prevBtn.style.setProperty('--hover-color', colorSet.hoverColor);
+  nextBtn.style.setProperty('--bg-color', colorSet.buttonColor);
+  nextBtn.style.setProperty('--shadow-bg', colorSet.shadowColor);
+  nextBtn.style.setProperty('--hover-color', colorSet.hoverColor);
+  button.style.setProperty('--bg-orange', colorSet.buttonColor);
+  button.style.setProperty('--shadow-bg', colorSet.shadowColor);
+  button.style.setProperty('--hover-color', colorSet.hoverColor);
```

Comment:
Solo para mejorar la lectura
```suggestion
  ellipse.style.setProperty('--bg-color', colorSet.ellipseColor);
  pagination.style.setProperty('--bg-orange', colorSet.buttonColor);
  price.style.setProperty('--bg-orange', colorSet.buttonColor);

  prevBtn.style.setProperty('--bg-color', colorSet.buttonColor);
  prevBtn.style.setProperty('--shadow-bg', colorSet.shadowColor);
  prevBtn.style.setProperty('--hover-color', colorSet.hoverColor);

  nextBtn.style.setProperty('--bg-color', colorSet.buttonColor);
  nextBtn.style.setProperty('--shadow-bg', colorSet.shadowColor);
  nextBtn.style.setProperty('--hover-color', colorSet.hoverColor);

  button.style.setProperty('--bg-orange', colorSet.buttonColor);
  button.style.setProperty('--shadow-bg', colorSet.shadowColor);
  button.style.setProperty('--hover-color', colorSet.hoverColor);
```